01-js-intro-and-jquery/js/javascript.js

```diff
@@ -0,0 +1,84 @@
+
+
+var functionFadeToggleSection = (function() {
+  //Private variables
+  var selectedSection = $('section.hidden-section');
+  var text = $('input.alias');
+
+  var callBackFunction = function() {
+    text.trigger('focus'); //only in chrome?
+    
+  };
+
+  var ft = function() {
```

Comment:
Again, try to use descriptive names (nobody except you will know what *"ft"* means, and maybe in a couple of days you'll forget it :P).